### DISCUSSION TITLE
Development On Mac?

### DISCUSSION THREAD

**Discussion by Krrishdhaneja** [Krrishdhaneja/virtual-assistants#20] [General]
Hey, @fm-sys we know that it is working fine on Windows, and on Linux but what about Mac, we need to assure that it works on Mac, so, do you know any dev, with a Mac who can test it out and if yes, it would be too great for our project!

**Comment by Krrishdhaneja** [comment]
Quick suggestion: How about asking for Mac development on dev.to , cuz that is a great community? What about this idea?

**Comment by fm-sys** [comment]
> Hey, @fm-sys we know that it is working fine on Windows, and on Linux but what about Mac, we need to assure that it works on Mac, so, do you know any dev, with a Mac who can test it out and if yes, it would be too great for our project!

Why does it need to be tested on Mac when there are no Mac users? 🤔

**Reply by fm-sys** [reply]
Better put your energy into improving Friday on Windows instead of targeting non-existing user groups...

**Reply by Krrishdhaneja** [reply]
> > Hey, @fm-sys we know that it is working fine on Windows, and on Linux but what about Mac, we need to assure that it works on Mac, so, do you know any dev, with a Mac who can test it out and if yes, it would be too great for our project!
> 
> Why does it need to be tested on Mac when there are no Mac users? 🤔

I was just making it sure that it works on every platform, cuz we don’t know that the next minute, who is gonna use it.

**Reply by Krrishdhaneja** [reply]
Hey @fm-sys , I don’t think so that I would be active on GitHub till mid February as my final exams are coming the next month.

**Reply by fm-sys** [reply]
In all likelihood, I don't either. Developing the Friday UI was like a winter holiday challenge for me (I hadn't coded in python for a very long time) but as I'm a actively maintaining multiple open sorce projects with quite large user groups and also for me school starts next week I will not have time anymore... Sorry!

**Reply by Krrishdhaneja** [reply]
No Problem!

**Reply by Krrishdhaneja** [reply]
Hey, @fm-sys you must update your fork, as I have provided info for Arch linux based systems in README.

**Reply by Krrishdhaneja** [reply]
Is anything resolved in snapdrop repo?
